firedrake/interpolation.py

```diff
@@ -1167,6 +1184,37 @@ def _interpolator(V, tensor, expr, subset, arguments, access, bcs=None):
         return copyin + callables + (parloop_compute_callable, ) + copyout
 
 
+def get_interp_node_map(source_mesh, target_mesh, fs):
+    """Return the cell-to-node map required by a parloop on the target_mesh.cell_set."""
```

Comment:
Can this be a full docstring with type hints? It isn't very clear as is.

Reply:
If I rename `fs` to `function_space`, then a docstring will not add any extra information

Reply:
Something like:
"Return the map between cells of the target mesh and nodes of the function space. If the function space is defined on the source mesh then the node map is composed with a map between target and source cells."

I'm not saying that it's impossible to understand, just that it could be clearer. It initially confused me.